### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by fur-gan**
Various ui improvements such as:
- Chipmunk now in dark-mode theme
- Keyboard shortcuts next to toolbar options
- New toolbar Option "Chipmunk"

### CODE REVIEW

**Comment by DmitryAstafyev** [comment]
Something weird

Screenshot 2021-03-18 at 09 49 14
Screenshot 2021-03-18 at 09 49 20

**Comment by DmitryAstafyev** [comment]
An application menu item (like menu item "Chipmunk") is using in macOS, but as for Linux and windows - doesn't not. It looks weird:

Screenshot from 2021-03-18 10-14-42

For example how VSCode looks like on linux

Screenshot from 2021-03-18 10-15-00

No application menu item - "Code". Menu starts from "File". But it has an analog of macOS app's menu item. It looks like it
 
Screenshot from 2021-03-18 10-15-04

Maybe we can use this.

**Comment by DmitryAstafyev** [comment]
I would close this pull request as soon as issue #1080 requires more specifications/details. We should describe our expectation for each platform as soon as it's different for mac/win/linux

File: application/electron/src/controllers/electron/controller.electron.menu.ts [hunk 1]
```diff
@@ -21,6 +21,10 @@ import * as os from 'os';
 const MAX_NUMBER_OF_RECENT_FILES = 20;
 
 const MENU_TEMPLATE = [
+    {
+        label: 'Chipmunk',
+        submenu: [ ],
+    },
```
Comment (bzld) [inline application/electron/src/controllers/electron/controller.electron.menu.ts:28]: This seems wrong. The existing code below already adds this if platform==darwin.

File: application/electron/src/controllers/electron/controller.electron.menu.ts [hunk 2]
```diff
@@ -126,15 +130,14 @@ export default class ControllerElectronMenu {
...
+                template[1].submenu.push({
                     label: action.caption,
                     click: HandlerItemExportActionCall.bind(null, action.id),
                 });
             });
         }
-        template[0].submenu.push({ type: 'separator' });
         template[0].submenu.push({
             label: 'Plugins',
             click: HandlerItemPlugins,
```
Comment (bzld) [inline application/electron/src/controllers/electron/controller.electron.menu.ts:145]: Move this inside the `if (darwin) {} else {}` block right below, and put it to main menu on darwin (right below Settings), and File menu on other platforms.

Comment (DmitryAstafyev) [reply application/electron/src/controllers/electron/controller.electron.menu.ts:145]: Hi Florian, 
We've closed this pull-request without merge as soon as original issue #1080 isn't clean. This issue (#1080) should be described in more detail considering each platform. We should have more or less the same UI and user expectations for all platforms, but issue was creater more to macos direction.

Partly it's already implemented. For example, recent documents are already available via the dock context menu (mac) and taskbar (windows).

**Pull request closed without merging**